discord/ext/converters/range.py

```diff
@@ -0,0 +1,13 @@
+from typing import Tuple
+
+from discord.ext import commands
+
+
+class InRangeConverter(commands.Converter):
+    def __init__(self, *args, **kwargs) -> None:
+        self.range = range(*args, **kwargs)
+
+    async def convert(self, ctx, argument: str) -> int:
+        if argument.isnumeric() and int(argument) in self.range:
```

Comment:
`str.isnumeric` does not mean `int(str)` will not fail. This should follow zen and ask forgiveness instead.